rts/motoko-rts/src/memory.rs

```diff
@@ -0,0 +1,35 @@
+#[cfg(feature = "ic")]
+pub mod ic;
+
+use crate::rts_trap_with;
+use crate::types::*;
+
+pub trait Memory {
+    unsafe fn alloc_words(&mut self, n: Words<u32>) -> SkewedPtr;
+
+    unsafe fn get_hp(&self) -> usize;
```

Comment:
We discussed this trait with Ulan and realized that this method is only used by the collectors and so should not be a part of this trait. I will remove this.